src/theme/Contexts/color.tsx

```diff
@@ -16,8 +16,32 @@ const Colors = {
 export type Color = (typeof Colors)[keyof typeof Colors]
 
 function useContextValue(): ContextValue {
+  const breakpoints = {
+    mobile: 1024
+  }
+  const mobileColor = Colors.light
+  const pcColor = Colors.dark
+
   // landing default color mode
-  const [color, setColorState] = useState<Color>(Colors.light)
+  // const [color, setColorState] = useState<Color>(Colors.dark)
+  const [color, setColorState] = useState<Color>(getColorMode(window.innerWidth))
+
+  function getColorMode(windowWidth: number): Color {
+    return windowWidth < breakpoints.mobile ? mobileColor : pcColor
+  }
+
```

Comment:
We could fix the errors in building documentations.
- From
```tsx
const [color, setColorState] = useState<Color>(getColorMode(window.innerWidth))
```

- To
```tsx
  const [color, setColorState] = useState<Color>()

  function getColorMode(windowWidth: number): Color {
    return windowWidth < breakpoints.mobile ? mobileColor : pcColor
  }

  useEffect(() => {
    const firstColor = getColorMode(window.innerWidth)
    setColorState(firstColor)
  }, [])
```

Maybe this framework doesn't know what `window` is when building apps.

Reply:
Okay I'll change this. Thank you.